UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/FulfilmentServiceJob.cs

```diff
@@ -89,7 +94,18 @@ public async Task CreateAndUploadErrorFileToFileShareService(SalesCatalogueServi
                     logger.LogError(EventIds.ErrorTxtNotUploaded.ToEventId(), "Error while uploading error.txt file to file share service for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}", fulfilmentServiceQueueMessage.BatchId, fulfilmentServiceQueueMessage.CorrelationId);
             }
             else
```

Comment:
Realise this isn't part of this pull request but is the code above correct?

if (isUploaded)
                {
                    logger.LogError(........

i.e. Looks like we are logging an error when we get a true result back from the UploadFileToFileShareService(........

Reply:
Since line number 86 is creating exchange set with error and error.txt, we are logging line 90 and 91 as error and not information so that we can come to know Exchange set is created with error.

Reply:
Thanks I think that tweak you have made makes it clearer. Appreciate the explanation!